data_license.go

```diff
@@ -328,6 +331,21 @@ var (
 			},
 			Prohibits: []string{},
 		},
+
+		IODLV2URL: DataLicense{
+			Name:         "Italian Open Data License v2.0",
+			URL:          "https://www.dati.gov.it/content/italian-open-data-license-v20",
+			LegalCodeURL: "https://www.dati.gov.it/content/italian-open-data-license-v20",
+			Permits: []string{
+				Reproduction,
+				Distribution,
+				DerivativeWorks,
```

Comment:
UK's OGL we had `Sharing` in the Permits array. I suppose this one is similar. I'll let @Popcore read the license page and compare this the CC ontology: https://creativecommons.org/ns and then he will decide

Reply:
@chespinoza and @dotmancando according to their definition (and the wikipedia one as well) the Italian Open Data License v2.0 is compatible with CC3.0 and [ODBL](https://opendatacommons.org/licenses/odbl/)

Reply:
our definition of OBDL has `Sharing` in Permits array. Shall we add `Sharing` to this license as well?

Reply:
I agree with @dotmancando we should have `Sharing` in the `Permits` slice since it is part of the license definition.

Reply:
@dotmancando @Popcore done.